packages/docusaurus-theme-openapi-docs/src/markdown/utils.ts

```diff
@@ -26,11 +26,11 @@ export function guard<T>(
   value: T | undefined | string,
   cb: (value: T) => Children
 ): string {
-  if (!!value) {
-    const children = cb(value as T);
-    return render(children);
+  if (value === undefined || value === "") {
```

Comment:
@sserrata this change introduced a bug in 3.0.0-beta.6, when `guard` is called from `SchemaItem/index.tsx` (for `renderRequired`, `value` is a `boolean`.
The check on this line evaluates to `false` and all `<SchemaItem>` are rendered as required.